juno/assetlist.json

```diff
@@ -185,6 +185,29 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/raw.png"
         },
         "coingecko_id": "junoswap-raw-dao"
+      },
+      {
+        "description": "Profit sharing token for Another.Software validator. Hold and receive dividends from Another.Software validator commissions!",
+        "type_asset": "cw20",
+        "address": "juno17wzaxtfdw5em7lc94yed4ylgjme63eh73lm3lutp2rhcxttyvpwsypjm4w",
+        "denom_units": [
+          {
+            "denom": "cw20:juno17wzaxtfdw5em7lc94yed4ylgjme63eh73lm3lutp2rhcxttyvpwsypjm4w",
+            "exponent": 0
+          },
+          {
+            "denom": "asvt",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno17wzaxtfdw5em7lc94yed4ylgjme63eh73lm3lutp2rhcxttyvpwsypjm4w",
+        "name": "Another.Software Validator Token",
+        "display": "asvt",
+        "symbol": "ASVT",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/asvt.png"
+        },
+        "coingecko_id": "anothersoftware-token"
```

Comment:
I don't see this on the CoinGecko's coins list:
https://api.coingecko.com/api/v3/coins/list

Reply:
I just left it because I don't know will it work without this field. As I understood token can't be listed on coingecko without trading pairs on exchanges. To add this coin to osmosis it needs firstly to be added in chain registry. Will it work if this line will be removed?

Reply:
Yes it will be fine without it for now, please remove it. but I look forward to getting it added once there is a valid coinGecko Id